server/py/framework/utils/auth/verifier.py

```diff
@@ -322,3 +320,51 @@ def _parse_basic_auth(header):
         b64value = header[len(AuthVerifier._basic_prefix) :]
         value = base64.b64decode(b64value).decode()
         return value.split(":", 1)
+
+    def _authenticate_basic(
+        self, headers: typing.Mapping[str, str]
+    ) -> mlrun.common.schemas.AuthInfo:
+        header = headers.get("Authorization", "")
+        if not header.startswith(self._basic_prefix):
```

Comment:
self._basic_prefix and  AuthVerifier._basic_prefix from _parse_basic_auth are the same value? Minor, but it would probably be better to have the usages aligned